pydis_site/apps/content/resources/guides/python-guides/vps-services.md

```diff
@@ -0,0 +1,35 @@
+---
+title: VPS Services
+description: On different VPS services
+---
+
+If you need to run your bot 24/7 (with no downtime), you should consider using a virtual private server (VPS).This is a list of VPS services that are sufficient for running Discord bots.
+
+* Europe
+    * [netcup](https://www.netcup.eu/)
+        * Germany & Austria data centres.
+        * Great affiliate program.
+    * [Yandex Cloud](https://cloud.yandex.ru/)
+        * Vladimir, Ryazan, and Moscow region data centres.
+    * [Scaleway](https://www.scaleway.com/)
+        * France data centre.
+    * [Time 4 VPS](https://www.time4vps.eu/)
+        * Lithuania data centre.
+* US
+    * [GalaxyGate](https://galaxygate.net/)
+        * New York data centre.
+        * Great affiliate program.
+* Global
+    * [Linode](https://www.linode.com/)
+        * Global data centre options.
+    * [Digital Ocean](https://www.digitalocean.com/)
+        * Global data centre options.
+    * [OVHcloud](https://www.ovhcloud.com/)
+        * Global data centre options.
+    * [Vultr](https://www.vultr.com/)
+        * Global data centre options.
+
+---
+Free hosts
```

Comment:
Perhaps make this a header rather than free text, so it shows what the next section is about and you can bookmark it?
```suggestion
# Free hosts
```